documentation/release_notes.markdown

```diff
@@ -109,131 +162,150 @@ stable release of PuppetDB, which included an experimental v4 API.
-  latest puppet run for each node.
-  ([PDB-1244](https://tickets.puppetlabs.com/browse/PDB-1244))
+* To improve performance on the `aggregate-event-counts` endpoint, we've modified the API and underlying storage indexing. We've also dropped HSQLDB support for this endpoint. ([PDB-1587](https://tickets.puppetlabs.com/browse/PDB-1587))
 
-* We have dropped the latest_reports table in favor of storing a reference to
-  the latest report of each certname in the certnames table.
-  ([PDB-1254](https://tickets.puppetlabs.com/browse/PDB-1254))
+#### Deprecations
 
-* The load testing tool can now optionally use a default set of testing data or
-derive data from a PuppetDB export tarball.
+* HyperSQL is now deprecated and will be removed in the next major release ([PDB-1508](https://tickets.puppetlabs.com/browse/PDB-1508))
 
-* We have now added [edges](./api/query/v4/edges.html) querying capability to the top-level namespace. You can also query edges specific to a particular catalog using the new `edges` suffix on the [catalogs](./api/query/v4/catalogs.html) endpoint. ([PDB-1228](https://tickets.puppetlabs.com/browse/PDB-954)).
+    Installing via the package will now no longer default to using HyperSQL.
+    Instead now we expect PostgreSQL by default, and will return an error if
+    it hasn't been configured.
 
-#### API changes
+### New Features
 
-* The query parameter for `/pdb/query/v4/events` is now optional ([PDB-1132](https://tickets.puppetlabs.com/browse/PDB-1132))
+* We now store report metrics ([PDB-1192](https://tickets.puppetlabs.com/browse/PDB-1192))
 
-* Add pagination to service query method ([PDB-1071](https://tickets.puppetlabs.com/browse/PDB-1071))
+* We now store report logs ([PDB-1192](https://tickets.puppetlabs.com/browse/PDB-1184))
 
-    Add a `paging_options` argument to the service query method.
+* We have added `count` aggregation capabilities to the query API ([PDB-1181](https://tickets.puppetlabs.com/browse/PDB-1181))
 
-* Make `/pdb/query/v4/catalogs` queryable ([PDB-1028](https://tickets.puppetlabs.com/browse/PDB-1028))
+* Support `extract` as a top level query operator ([PDB-207](https://tickets.puppetlabs.com/browse/PDB-207))
 
-* Add `producer_timestamp` field to factsets and catalogs ([PDB-489](https://tickets.puppetlabs.com/browse/PDB-489))
+    This provides support for requesting only the fields that you need in
+    an API response when querying data.
 
-    On factsets the field is queryable and orderable.
+* Support fallback PuppetDB connections ([PDB-100](https://tickets.puppetlabs.com/browse/PDB-100))
 
-* Add `resource_events` key to the `store report` command ([PDB-1072](https://tickets.puppetlabs.com/browse/PDB-1072))
+    When configuring your connection to PuppetDB, you can now specify a fallback.
+    Instead of the old `server` and `port` config, you can now use a
+    `server\_urls` config that contains the full path to one or more PuppetDB
+    URLs. The old server/port format is still supported.
```

Comment:
It looks like we still support it [here](https://github.com/puppetlabs/puppetdb/blob/stable/puppet/lib/puppet/util/puppetdb/config.rb#L26). Is there something I'm missing?